packages/esm-patient-search-app/src/root.component.tsx

```diff
@@ -4,9 +4,9 @@ import PatientSearchPageComponent from './patient-search-page/patient-search-pag
 
 const PatientSearchRootComponent: React.FC = () => {
   return (
-    <BrowserRouter basename={`${window['getOpenmrsSpaBase']()}search`}>
+    <BrowserRouter basename={window.getOpenmrsSpaBase()}>
       <Routes>
-        <Route path="" element={<PatientSearchPageComponent />} />
+        <Route path="/search" element={<PatientSearchPageComponent />} />
```

Comment:
Shouldn't this be `"search"`? (`getOpenmrsSpaBase()` returns `/openmrs/spa/`)

Reply:
It should! Thanks